deployment/live/gcp/static-ct-staging/cloudbuild/README.md

```diff
@@ -0,0 +1,23 @@
+# GCP Cloud Build Triggers and Steps
+
+This directory contains terragrunt files to configure our Cloud Build pipeline(s).
+
+The Cloud Build pipeline is triggered when a commit in the repo is tagged with
+`deploy-staging-XX` and is responsible for:
```

Comment:
Is there a pattern of `XX`? Is it a timestamp or an unsigned incremental integer?